python/pyspark/context.py

```diff
@@ -1006,6 +1011,33 @@ def getConf(self):
         conf.setAll(self._conf.getAll())
         return conf
 
+    def _get_executor(self):
```

Comment:
Given that we use executors to mean things on the worker this is super confusing name, albiet a reasonable name in another system.